internal/pkg/convert/graph.go

```diff
@@ -107,15 +107,17 @@ func (graph *GraphLLB) buildBaseImages() {
 }
 
 func (graph *GraphLLB) buildChecksummer() {
+	// use reproducible stagex images to get `sha512sum` binary
+	const stagexRelease = "sx2024.12.0"
```

Comment:
let's move this to constants and add renovate comment

Reply:
I don't think we really need to renovate this? as it's nothing we need new features for ;) but I turned into a draft for now

Reply:
resolved, moved to constants